x-pack/plugin/runtime-fields/src/test/java/org/elasticsearch/xpack/runtimefields/mapper/ScriptDateMappedFieldTypeTests.java

```diff
@@ -373,4 +430,9 @@ private void checkExpensiveQuery(BiConsumer<ScriptDateMappedFieldType, QueryShar
             equalTo("queries cannot be executed against [runtime_script] fields while [search.allow_expensive_queries] is set to [false].")
         );
     }
+
+    private void checkBadDate(ThrowingRunnable queryBuilder) {
+        Exception e = expectThrows(ElasticsearchParseException.class, queryBuilder);
+        assertThat(e.getMessage(), containsString("failed to parse date field"));
+    }
```

Comment:
would it make sense to have some duel tests between concrete dates and runtime dates to make sure that formatting and locale are applied the same way?

Reply:
Yeah!